### PR TITLE
feat(iam): adds lens default profile stamp

### LINKED ISSUE #432: [IAM] As an IAM server, I want to issue stamps for a registered Lens profile

### PR DESCRIPTION
**Pull request by farque65**
Closes #432

### CODE REVIEW

File: iam/src/providers/lens.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+  // Options can be set here and/or via the constructor
+  _options = {};
+
+  // construct the provider instance with supplied options
+  constructor(options: ProviderOptions = {}) {
+    this._options = { ...this._options, ...options };
+  }
+
+  // Verify that address defined in the payload owns at least one POAP older than 15 days
```
Comment (nutrina) [inline iam/src/providers/lens.ts:82]: Please adjust or remove this comment

File: iam/src/providers/lens.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+  const profileId: unknown = await contract.defaultProfile(userAddress);
+
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+  const userHandle: unknown = await contract.getHandle(profileId);
+
+  return userHandle?.toString();
+}
+
+// Export a POAP Provider
```
Comment (nutrina) [inline iam/src/providers/lens.ts:69]:
```suggestion
// Export a LensProfileProvider
```

**Review by nutrina — CHANGES_REQUESTED** [review]
Just some minor changes

**Review by aminah-io — COMMENTED** [review]
Just one comment!

**Review by aminah-io — APPROVED** [review]
LGTM! 🚀

**Pull request merged**